UnitTests/GitCommands.Tests/Git/GetAllChangedFilesOutputParserTest.cs

```diff
@@ -1,44 +1,39 @@
+        [TestCase("both_added", "u AA N... 000000 100644 100644 100644 0000000000000000000000000000000000000000 61780798228d17af2d34fce4cfbdf35556832472 78981922613b2afb6025042ff6bd878ac1994e85 t.t")]
+        [TestCase("both_deleted", "u DD N... 100644 000000 000000 000000 3c70853f1ed9b82635f0763dd1373c3101e79d5d 0000000000000000000000000000000000000000 0000000000000000000000000000000000000000 GitUI/CommandsDialogs/FormFormatPatch.cs")]
+        [TestCase("added_by_us", "u AU N... 000000 100644 000000 100644 0000000000000000000000000000000000000000 3c70853f1ed9b82635f0763dd1373c3101e79d5d 0000000000000000000000000000000000000000 a2")]
+        [TestCase("added_by_them", "u UA N... 000000 000000 100644 100644 0000000000000000000000000000000000000000 0000000000000000000000000000000000000000 3c70853f1ed9b82635f0763dd1373c3101e79d5d b2")]
+        [TestCase("deleted_by_them", "u UD N... 100644 100644 000000 100644 f7f9ab7c975ab68e9bd8bda93b8bb717987947ea bf42fd75847fdc76d4f4a2b89d92a1b289b2e4ea 0000000000000000000000000000000000000000 GitUI/CommandsDialogs/FormFormatPatch.Designer.cs")]
+        [TestCase("deleted_by_us", "u DU N... 100644 000000 100644 100644 dc7a0a8364df7cb022b3e29a48ecd84992af4613 0000000000000000000000000000000000000000 bf42fd75847fdc76d4f4a2b89d92a1b289b2e4ea GitUI/CommandsDialogs/FormFormatPatch.Designer.cs")]
 
         // Note that it is not expected that fatal: is mixed with proper info, but parsing should handle
-        [TestCase("fatal error",
+        [TestCase("fatal_error",
     "? unknown info\0fatal: bad config line 1 in file F:/dev/gc/gitextensions/.git/modules/GitExtensionsDoc/config\nfatal: 'git status --porcelain=2' failed in submodule GitExtensionsDoc\n1 MM N... 160000 160000 160000 cbca134e29be13b35f21ca4553ba04f796324b1c cbca134e29be13b35f21ca4553ba04f796324b1c adfs.h\0")]
-        public void TestGetStatusChangedFilesFromString(string testName, string statusString)
+        public async Task TestGetStatusChangedFilesFromString(string testName, string statusString)
         {
             // TODO produce a valid working directory
             GitModule module = new(Path.GetTempPath());
             GetAllChangedFilesOutputParser getAllChangedFilesOutputParser = new(() => module);
 
-            using (ApprovalResults.ForScenario(testName.Replace(' ', '_')))
-            {
-                // git status --porcelain=2 --untracked-files=no -z
-                var statuses = getAllChangedFilesOutputParser.Parse(statusString);
-                Approvals.VerifyJson(JsonConvert.SerializeObject(statuses));
-            }
+            // git status --porcelain=2 --untracked-files=no -z
+            IReadOnlyList<GitItemStatus> statuses = getAllChangedFilesOutputParser.Parse(statusString);
+            await Verifier.VerifyJson(JsonConvert.SerializeObject(statuses))
```

Comment:
@simoncropp could you give me a hint whether I missed something here? ApprovalTests produced "proper" json, and Verify appears to produce json-like output. Is there a way to retain the proper json format? And is there a reason why `VerifyJson` uses .txt instead of .json for snapshots?
Thanks

Reply:
can u try UseStrictJson https://github.com/VerifyTests/Verify/blob/main/docs/serializer-settings.md#usestrictjson

Reply:
Thanks for the tip, I'll give it a try.